distance-calculator/charts/mqttwriter/templates/mqttwriter.yaml

```diff
@@ -0,0 +1,25 @@
+{{- $mqtt := .Values.mqtt -}}
+apiVersion: v1
+kind: Pod
```

Comment:
@Sanjay1611 , @Bilna are we planning on deploying mqtt writer as a pod too ? or do we use it as a client / binary outside SDP to push simulated/real data from local file. 
I understand that both of these are feasible solutions , but we might need to be clear on what direction the deliverable has to be for helm charts. (All deployable modules / deployable helm modules( mqtt broker , flink cluster,app , grafana ) +external ingestion clients (mqtt writer)

Reply:
We can deploy mqtt-writer as a pod

Reply:
@Vi-Nk Thats what was not clear in the story i.e. upto what extent automation has to be provided. But we did not hand to engage into a discussion. So I think for now running it as a pod should be fine ... we can modify the charts as per the requirement